Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -497,6 +587,32 @@ codeunit 4022 "GP Vendor Migrator"
         until GPSY06000.Next() = 0;
     end;
 
+    local procedure IsValidIBANCode(IBANCode: Code[100]): Boolean
```

Comment:
This is a "pre-validation" check, because the IBAN validation trigger does indeed use CompanyInfomration.CheckIBAN, and it will throw an error if the check fails. The fix here is to not go through the full validation unless the code is fairly confident that the IBAN will pass the full validation.